integration/sawtooth_integration/docker/xo-smoke.yaml

```diff
@@ -23,8 +23,9 @@ services:
       - ../../../:/project/sawtooth-core
     expose:
       - 40000
-    entrypoint: "bash -c \"\
-        ./bin/tp_config -v validator:40000 \""
+    depends_on:
+      - validator
+    entrypoint: ./bin/tp_config -vv validator:40000
```

Comment:
I think so. Until we have tar files of the logs, it will help debugging failed tests.